src/pymovements/utils/parsing.py

```diff
@@ -29,7 +29,7 @@
 import polars as pl
 
 EYE_TRACKING_SAMPLE = re.compile(
-    r'(?P<time>(\d{7,10}))\s+'
+    r'(?P<time>(\d{6,10}))\s+'
```

Comment:
is it possible to go even lower (or higher) here, like `\d{4, 20}` or just a `\d+`? I don't have the EyeLink documentation at hand, and I don't really know where these ranges came from.

Reply:
I quickly checked the manual and it does not say anthing specific about the length. But I assume it won't be lower than 3 or 4 as this would mean that the first timestamp is within 1 second of the experiment start.. But the manual says that only sample lines begin with digits so we are probably safe if we use `\d+`